test/test_dynamic_shapes.py

```diff
@@ -4,25 +4,25 @@
 import torch, torch_xla
 import torch_xla.core.xla_model as xm
 
+from torch._dispatch.python import enable_python_dispatcher
+
 dev = xm.xla_device()
 
 
 class TestDynamicShapes(unittest.TestCase):
 
-  @unittest.skip("fails with functionalization")
   def test_simple_expand(self):
-    size1 = 5
-    size2 = 2
-    t1 = torch.zeros([size1, size2], device=dev)
-    t1[3][0] = 1
-    t1[3][1] = 1
-    # t2 has size [<=10, 2]
-    t2 = torch.nonzero(t1)
-    t5 = torch.ones(1, device=dev)
-    t6 = t5.expand(t2.size(0))
-    self.assertIn('<=10', torch_xla._XLAC._get_xla_tensors_text([t6]))
```

Comment:
Any reason to remove the check `self.assertIn('<=10', torch_xla._XLAC._get_xla_tensors_text([t6]))`? It's to check `XLASymNodeImpl::str()` method.

Reply:
Oh, good catch. It could be just rebase error...